projectDocs/dev/codingStandards.md

```diff
@@ -149,6 +149,87 @@ Type hints make reasoning about code much easier, and allow static analysis tool
 * Prefer union shorthand (`X | Y`) over explicitly using `typing.Union`.
```

Comment:
Can you add a small note to changes for developers introducing this new module and referring to the developer guide for more information

Reply:
done